lib/ansible/modules/cloud/amazon/aws_kms.py

```diff
@@ -33,14 +33,23 @@
     required: true
     default: grant
     choices: [ grant, deny ]
-  key_alias:
-    description:
-    - Alias label to the key. One of C(key_alias) or C(key_arn) are required.
+  alias:
+    description: An alias for a key. For safety, even though KMS does not require keys
+      to have an alias, this module expects all new keys to be given an alias
+      to make them easier to manage. Existing keys without an alias may be
```

Comment:
This isn't true, is it? The "For safety" sentence- because it's basically "alias is required.. unless not, which is okay".